Kickstarter-iOS/Views/Cells/RewardCell.swift

```diff
@@ -219,28 +274,26 @@ final class RewardCell: UICollectionViewCell, ValueCell {
     _ = ([self.priceStackView, self.stateImageViewContainer], self.titleStackView)
       |> ksr_addArrangedSubviewsToStackView()
 
-    self.setupConstraints()
-
-    self.pledgeButton.addTarget(self, action: #selector(self.pledgeButtonTapped), for: .touchUpInside)
-
-    let tapGestureRecognizer = UITapGestureRecognizer(target: self, action: #selector(self.rewardCardTapped))
-    self.containerView.addGestureRecognizer(tapGestureRecognizer)
+    self.pillCollectionView.registerCellClass(PillCell.self)
```

Comment:
Because `PillCell` conforms to `ValueCell` you should be able to use `self.pillCollectionView.register(PillCell.self)`, which is what we use for other `ValueCell`- conforming cells. It's confusing that we have two and I think we could refactor these helpers so we only have one way to register cell that works both with `ValueCellDataSource` and without it, but I just wanted to flag that `ValueCellDataSource` relies on the cells being registered with `T.defaultReusableId` as the identifier.